flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/BuiltInFunctionTestBase.java

```diff
@@ -193,7 +193,6 @@ private static void testError(
         } catch (AssertionError e) {
             throw e;
         } catch (Throwable t) {
-            assertTrue(t instanceof ValidationException);
```

Comment:
Just wonder why should we stop checking for ValidationException here?

Reply:
Because a TableException may also be thrown here when we check the invalid implicit type conversion between numeric and (var)char.  I think writing another catch like `catch (TableException e ) ... ` is unnecessary and I can reuse 　the code here.

Reply:
Yes, however it seems it was introduced with a purpose for BuiltInFunction tests. I'm not sure if it is ok or not to change it for all BuiltIn function tests.
What do you think @twalthr ?